inc/class-command.php

```diff
@@ -151,4 +151,18 @@ public function jobs( $args, $assoc_args  ) {
 		}
 
 	}
+
+	/**
+	 * Upgrade to the latest database schema.
+	 */
+	public function upgrade( ) {
+		require_once __DIR__ . '/upgrade/namespace.php';
+
+		if ( Upgrade\upgrade_database() ) {
+			\WP_CLI::success( 'Database version upgraded.' );
+			return;
+		}
+
+		\WP_CLI::error( 'Databse upgrade not required.' );
```

Comment:
`Database`. Also, IMO this should be `::success` instead, unless there's a good reason to fail. Given there's no `needs-upgrade` command, you otherwise can't know beforehand if this will fail or not.

Reply:
Spellng corrected, switched to success in 9bf2bdbe149212b1f27454442c92277822568e66